src/test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs

```diff
@@ -50,6 +50,12 @@ public void Running_Build_Output_Standalone_EXE_with_DepsJson_and_RuntimeConfig_
 
             var appExe = fixture.TestProject.AppExe;
 
+            // TODO: Use FS.Chmod when build utility project is converted to csproj.
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                Command.Create("chmod", "+x", appExe).Execute().EnsureSuccessful();
```

Comment:
I think just `chmod u+x` will suffice.  The whole world won't need to execute this file.  Just the owner.